site/README-JEKYLL.md

```diff
@@ -31,3 +31,25 @@ This mirrors the plug-ins used by GitHub Pages on your local machine including J
 7. View your website at http://127.0.0.1:4000/
 8. Commit any changes and push everything to your fork.
 9. Once you're ready, submit a PR of your changes. Netlify will automatically generate a preview of your changes.
+
+
+# Adding a New Docs Version
+
+To add a new set of versioned docs to go with a new Velero release:
+
+1. In `$VELEROROOT`, run:
+
+   ```bash
+   # set to the appropriate version number
+   NEW_DOCS_VERSION=vX.Y.Z make gen-docs
+   ```
+
+1. In `site/_config.yml`, under the `defaults` field, add an entry for the new version just under `master` by copying the most recent version's entry and updating the version numbers.
+
+1. Also in `site/_config.yml`, update the `latest` field to have the new version number as its value.
+
+1. Also in `site/_config.yml`, add the new version to the `versions` field, just under `- master`.
```

Comment:
I think `sed "/- master/a - ${NEW_DOCS_VERSION}" site/_config.yml` will do this

The "a" directive in the middle is a sed append operation.